### ISSUE TITLE
Adding evan.burnett@coforma.io to DSVA Slack

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#56371]
# Description

Need to add Evan Burnett from Coforma to our Slack. He will be helping out with accessibility work! 

Please add evan.burnett@coforma.ios a full member.

What channel(s) do they need to be added to?
- #accessibility-ops
- #accessibility-help 
- #sitewide-content-ia
(We will add him to more as needed)

Then add a brief description of this person, their role on the project, and why they should be added under the status you have requested.

- Evan will be helping with accessibility documentation and other requests in coordination with @coforma-terry
